integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -2,22 +2,19 @@ package smoke
 
 import (
 	"context"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
 	"github.com/smartcontractkit/chainlink/integration-tests/actions/vrfv2plus"
 	"github.com/smartcontractkit/chainlink/integration-tests/actions/vrfv2plus/vrfv2plus_constants"
+	"github.com/smartcontractkit/chainlink/integration-tests/docker/test_env"
 	"math/big"
 	"testing"
 	"time"
 
-	"github.com/onsi/gomega"
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
 	"github.com/stretchr/testify/require"
-
-	"github.com/smartcontractkit/chainlink/integration-tests/actions"
-	"github.com/smartcontractkit/chainlink/integration-tests/docker/test_env"
-	"github.com/smartcontractkit/chainlink/integration-tests/types/config/node"
 )
 
-func TestVRFv2PlusBasic(t *testing.T) {
```

Comment:
Does this test no longer exist then?

Reply:
yes, it is replaced with 2 tests (check link billing and native billing) which can be considered as 2 main base tests